policies/releasestrat.html

```diff
@@ -88,6 +88,10 @@ <h5>
 	  fixes. Before that, bug and security fixes will be applied
 	  as appropriate.</p>
 
+          <p>The addition of new platforms to LTS branches is acceptable so
+          long as the required changes consist solely of additions to
+          configuration.</p>
+
```

Comment:
Indentation is a little odd

Reply:
I used space, the original uses tabs.  It looks fine in an editor.